### PR TITLE
[fix] next_series_seasons - Limit size of emit in one run

### PR DESCRIPTION
**Pull request by tubedogg**
### Motivation for changes:
Given a season identifier of S2014 and backfill enabled, it is possible for `next_series_seasons` to emit all the way down to S01 in one run.

### Detailed changes:
This change prevents it from emitting if `begin` is not set and the group of seasons to review is more than 15, or if `begin` is set and the difference between the latest season to review and `begin` is more than 30. In both cases the user will be required to set or adjust `begin` before any emit will occur.

### Addressed issues:
- Fixes issue originally raised by @jawilson [here](https://github.com/Flexget/Flexget/commit/30751df7da16bdc12781411cdec9f51d9e02a890#commitcomment-22161686).

### CODE REVIEW

File: flexget/plugins/input/next_series_seasons.py [hunk 1]
```diff
@@ -115,6 +115,19 @@ def on_task_input(self, task, config):
                 else:
                     latest_season = low_season + 1
 
+                if (latest_season - low_season > 15 and not series.begin) or (series.begin and
```
Comment (liiight) [inline flexget/plugins/input/next_series_seasons.py]: Maybe move these values to consts?

File: flexget/plugins/input/next_series_seasons.py [hunk 2]
```diff
@@ -115,6 +115,19 @@ def on_task_input(self, task, config):
...
+                        log.error('Series `%s` has a begin episode set (`%s`), but the season currently being processed '
+                                  '(%s) is %s seasons later than it. To prevent emitting incorrect seasons, this ' 
+                                  'series will not emit unless the begin episode is adjusted to a season that is less '
+                                  'than 30 seasons from season %s.', series.name, series.begin.identifier, latest_season,
+                                  (latest_season - series.begin.season), latest_season)
+                    else:
+                        log.error('Series `%s` does not have a begin episode set and continuing this task would result '                                   'in more than 15 seasons being emitted. To prevent emitting incorrect seasons, this '
+                                  'series will not emit unless the begin episode is set in your series config or by '
+                                  'using the CLI subcommand `series begin "%s" <SxxExx>`.', series.name, series.name)
+                    continue
```
Comment (liiight) [inline flexget/plugins/input/next_series_seasons.py]: So if season is 2017 the user will get 2005 error messages in the log?

Comment (tubedogg) [inline flexget/plugins/input/next_series_seasons.py]: No, this is outside the loop that counts through the seasons. This should only happen once per series.

File: flexget/plugins/input/next_series_seasons.py [hunk 3]
```diff
@@ -71,6 +71,8 @@ def on_task_input(self, task, config):
         if isinstance(config, bool):
             config = {}
 
+        DIFF_NO_SERIES_BEGIN = 15
```
Comment (liiight) [inline flexget/plugins/input/next_series_seasons.py]: These should ideally go at the beginning of the module, outside if the plugin class. The idea is that constant values like these can be easily viewed when editing the file and kit hidden internally. 
Also I'm not too wild about their names but can't think of a better idea...

**Pull request merged**